tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py

```diff
@@ -52,20 +64,57 @@ def tap_first_received_message(self):
         from gaiatest.apps.messages.regions.message_thread import MessageThread
         return MessageThread(self.marionette)
 
+    def tap_thread_options(self):
+        self.marionette.find_element(*self._thread_options_button).tap()
+        from gaiatest.apps.messages.regions.options import ThreadOptions
+        return ThreadOptions(self.marionette)
+
+    def enter_select_mode(self):
+        edit_header = self.marionette.find_element(*self._edit_thread_header)
+        thread_options = self.tap_thread_options()
+        thread_options.select_threads()
+        Wait(self.marionette).until(lambda m: edit_header.rect['y'] == 0)
```

Comment:
I'll add it, to make the intent clearer. FYI, the header is at y < 0, when it's hidden